tests/multipleBackend/objectPutPart.js

```diff
@@ -109,15 +110,12 @@ errorDescription) {
                 uploadId: testUploadId,
             },
         };
-        if (partLoc) {
-            partReqParams.headers = { 'host': `${bucketName}.s3.amazonaws.com`,
-                'x-amz-meta-scal-location-constraint': `${partLoc}`,
-            };
-        }
         const partReq = new DummyRequest(partReqParams, body1);
         return objectPutPart(authInfo, partReq, undefined, log, err => {
             assert.strictEqual(err, null);
-            if (bucketLoc !== awsLocation && mpuLoc !== awsLocation) {
+            if (bucketLoc !== awsLocation && mpuLoc !== awsLocation &&
+            bucketLoc !== awsLocationMismatch &&
+            mpuLoc !== awsLocationMismatch) {
```

Comment:
what happens if it is an awsLocation/awsLocationMisMatch?

Reply:
We do not save the metadata of the object part.